src/Uno.UI.RemoteControl.VS/EntryPoint.cs

```diff
@@ -344,6 +345,7 @@ private async Task EnsureServerAsync()
 
 				_ideChannelClient = new IdeChannelClient(pipeGuid, new Logger(this));
 				_ideChannelClient.ForceHotReloadRequested += OnForceHotReloadRequestedAsync;
+				_ideChannelClient.InfoBarNotificationRequested += OnInfoBarNotificationRequestedAsync;
```

Comment:
I would have create a generic event `OnMessageReceived`, but I'm fine to keep it this way for now.

Reply:
Done. Updated. https://github.com/unoplatform/uno/commit/fa522e3b62af77d4c097808634ac8a9fbe085efa